daml/Marketplace/Utils.daml

```diff
@@ -40,3 +41,12 @@ getAccountOwnerProvider: Text -> (Party, Party)
 getAccountOwnerProvider label =
   let [owner, provider] = map (fromSome . partyFromText . replace "'" "") (splitOn "@" label)
   in (owner, provider)
+
+getDepositQuantities : [ContractId AssetDeposit] -> Update (Numeric 10)
+getDepositQuantities depositCids = do
+  deposit_quantities <- do
```

Comment:
Camel case should be used here as well, also this `do` is redundant

You could also drop the variable/initial do block altogether and do `foldl (+) 0.0 <$> forA depositCids ...etc`